src/main/java/org/broadinstitute/hellbender/tools/walkers/mutect/MitochondrialCallerArgumentCollection.java

```diff
@@ -0,0 +1,57 @@
+package org.broadinstitute.hellbender.tools.walkers.mutect;
+
+import org.broadinstitute.barclay.argparser.Advanced;
+import org.broadinstitute.barclay.argparser.Argument;
+import org.broadinstitute.hellbender.tools.walkers.haplotypecaller.AssemblyBasedCallerArgumentCollection;
+
+public class MitochondrialCallerArgumentCollection extends AssemblyBasedCallerArgumentCollection {
+    private static final long serialVersionUID = 9341L;
+    public static final String EMISSION_LOD_LONG_NAME = "lod-to-emit";
+    public static final String EMISSION_LOG_SHORT_NAME = "emit-lod";
+    public static final String INITIAL_LOD_LONG_NAME = "initial-lod";
+    public static final String INITIAL_LOD_SHORT_NAME = "init-lod";
+    public static final String MEDIAN_AUTOSOMAL_COVERAGE_LONG_NAME = "median-autosomal-coverage";
+
+    /**
+     * Only variants with LODs exceeding this threshold will be written to the VCF, regardless of filter status.
+     * Set to less than or equal to lod. Increase argument value to reduce false positives in the callset.
+     * Default setting of 0 is optimized for high sensitivity and will emit negative training data that
+     * {@link FilterMitochondrialCalls} should then filter.
+     */
+    @Argument(fullName = EMISSION_LOD_LONG_NAME, shortName = EMISSION_LOG_SHORT_NAME, optional = true, doc = "LOD threshold to emit variant to VCF.")
+    public double emissionLod = 0;
```

Comment:
FYI there is hopefully a very big improvement to M2 lod filtering coming soon that we would probably want to apply to mitochondrial calling.  Separating the tools and their argument collections will make it more cumbersome to bring in this change.  I'm not sure there's a good solution. . .